our-process/policies.md

```diff
@@ -22,31 +22,34 @@ When submitting a package, please make sure that your GitHub notification settin
 
 ## Submitting your package for review in other venues
 
-We strongly suggest submitting your package for review with pyOpenSci before:
+We recommend submitting your package for review with pyOpenSci before:
 
-*  publishing on `PyPI` or `conda`;
-*  submitting a software paper describing the package to a journal.
+- publishing to a package repository, such as `PyPI` and `conda-forge`;
+- submitting a software paper describing the package to a journal.
 
 Review feedback may result in major improvements and updates to your package,
 including changes that could be break package functionality.
 
-We do not consider previous publication on `PyPI` or `conda` or in other venues
+We do not consider previous publication on `PyPI` or `conda-forge` or in other venues
 sufficient reason to not adopt reviewer or editor recommendations provided in
 our review.
 
->Please do not submit your package for review while it or an associated manuscript is
+> Please do not submit your package for review while it or an associated manuscript is
 > also under review at another venue, as this may result on conflicting requests
 > for changes from two sets of reviewers.
 
 ### Publication with Journal of Open Source Software (JOSS)
+
 If you have previously published your software package with JOSS, you can still
 submit it to pyOpenSci for review. This provides:
 
-* Increased visibility of your package as a vetted tool within the scientific python ecosystem
-* We will also keep in touch with you as a maintainer to support long term maintenance. If you need to step down from maintaining your package we will help find a new maintainer and/or help sunset the tool.
+- Increased visibility of your package as a vetted tool within the scientific python ecosystem
+- We will also keep in touch with you as a maintainer to support long term maintenance. If you need to step down from maintaining your package we will help find a new maintainer and/or help sunset the tool.
 
 (coi)=
```

Comment:
Does this have meaning or is it stray text?

Reply:
oh @willingc i didn't understand this question but now i do

`(coi)=` is a [myst target](https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#creating-explicit-targets). it allows me to create a short form link to the target location within the book rather than using a long header name. so i think this is a conflict of interest section target that i created.